File: B10-CS-2023/Day 2 - Second Half/Staircase.py
'''
   #  1  3spaces 1star    (N-I)sp I*
  ##  2  2spaces 2stars
 ###  3  1space  3stars
####  4  0space  4stars
N=4
'''
N=int(input())
for I in range(1,N+1):
    print(' ' * (N-I), '*' * I, sep='')

File: B10-CS-2023/Day 2 - Second Half/Staircase.c
#include<stdio.h>
int main()
{
	int N;
	scanf("%d",&N);
	
	char spaces[] = "                                              ";
	char stars[] = "***********************************************";
	
	//printf("%.5s\n",stars);
	//printf("%.*s",7,stars);
	for(int I = 1; I <= N; I++)
		printf("%.*s%.*s\n",(N-I),spaces,I,stars);
	
	return 0;
}